### PR TITLE
Give a hint that the projects list is empty

### PR DESCRIPTION
**Pull request by suricactus**
image

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2175 Give indication there are no projects in the project list](https://app.clickup.com/t/2192114/QF-2175)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for a6a3e2e7df503e73a159e0ba1d83826201357268: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3293-93ce1001ef047d4449ebb6d24a48127a18d9c473-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3293-93ce1001ef047d4449ebb6d24a48127a18d9c473-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3293-93ce1001ef047d4449ebb6d24a48127a18d9c473-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3293-93ce1001ef047d4449ebb6d24a48127a18d9c473-x86-android.apk)
</details>

**Comment by m-kuhn** [comment]
Nice, could we make the text grey and I think @nirvn proposed the tooltip styling

File: src/qml/QFieldCloudScreen.qml [hunk 1]
```diff
@@ -387,6 +387,15 @@ Page {
                 }
             }
 
+            Label {
+                anchors.fill: parent
+                horizontalAlignment: Qt.AlignHCenter
+                verticalAlignment: Qt.AlignVCenter
+                visible: parent.count == 0
+                text: qsTr("No projects found")
+                font.bold: true
```
Comment (nirvn) [inline src/qml/QFieldCloudScreen.qml:396]:
```suggestion
                font: Theme.strongTipFont
                color: Theme.gray
```

**Comment by nirvn** [comment]
@suricactus , good UI improvement.

**Comment by m-kuhn** [comment]
Let's schedule that for one patch release later to give some time for translators to have a localized string

**Review by nirvn — APPROVED** [review]

**Pull request merged**